### DISCUSSION TITLE
无线必须手动 dns

### DISCUSSION THREAD

**Discussion by lesnolie** [kkkgo/PaoPaoGateWay#97] [问和答] [answered]
ppdns+ppgw 使用者。

今天发现遇到了问题，不知从什么时候开始，无线网络必须手动 DNS。

但是这个手动 DNS 和自动的 DNS 是一样的，加入自动分配的是 192.168.50.1，我必须改成自动，也填这个才能从 ppdns 分流到 ppgw

电视无论手动自动都无法使用 ppgw

**Comment by kkkgo** [comment]
检查下是否有被分配IPv6相关地址

**Reply by lesnolie** [reply]
不好意思竟然发现自己打了这么多错别字
有被分配 ipv6 的 dns
有线可以自动 dhcp
无线必须手动 dns，填ppdns 的 ipv4 地址。

请问是因为无线被分配了 ipv6dns 的原因吗？
如何解决呢

感谢！

**Reply by kkkgo** [reply]
大多数支持IPv6的现代系统，当存在可用的IPv6 DNS的时候会被优先使用。最简单的解决方法就是禁止IPv6 DNS下发（具体设置方式看你路由器系统），或者指定为ppdns的局域网v6地址。

**Reply by lesnolie** [reply]
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/81410185/17c25486-b767-472f-ba54-8a1d28deb6bd)
我要把这个自动接上IPV6地址给关了吗？
另外下发IPV6 DNS关闭以后设备还能使用IPV6吗？我需要IPV6来进行TAILSCALE 打洞

**Reply by kkkgo** [reply]
IPv6 DNS并非是使用IPv6的必要条件。IPv4 DNS可以完全替代IPv6 DNS的作用提供AAAA等解析记录。

**Reply by lesnolie** [reply]
是不是装个 adguardhome 也可以？

**Reply by kkkgo** [reply]
adguardhome无法阻止你的路由器下发IPv6 DNS。你需要搜索你的路由器关闭IPv6 DNS的方式。

**Reply by kkkgo** [reply, marked as answer]
也许这个连接对你有帮助： https://www.snbforums.com/threads/asuswrt-merlin-serving-ipv6-router-ip-instead-of-ipv6-dns-server-ip-f-w-384-19.67225/page-3

**Reply by lesnolie** [reply]
修好了，把图里的广播关了就可以了

**Reply by kkkgo** [reply]
按字面意思理解这个选项应该是Router Advertisement，禁用他可能会导致你无法获取分配IPv6地址。

**Reply by lesnolie** [reply]
您说的对，我尝试一下您上面发的地址

**Reply by lesnolie** [reply]
> merlin

完美
